discovery-provider/src/queries/get_trending_tracks.py

```diff
@@ -68,7 +68,11 @@ def generate_unpopulated_trending_from_mat_views(
         top_listen_tracks_subquery = top_listen_tracks_subquery.order_by(
             desc(AggregateIntervalPlay.year_listen_counts)
         )
-    top_listen_tracks_subquery = top_listen_tracks_subquery.limit(limit)
+
+    score_params = strategy.get_score_params()
```

Comment:
oh yes, the net multiplier is good

Reply:
do we need the net multiplier at all though? can we issue this request without a top listen subquery?

Reply:
in line w/ the "one experiment" thing

Reply:
This top listen tracks with the net multiplier is strictly to keep the trending consistent with the current trending algorithm. 
The current trending gets the top X (100* net_multiplier) tracks and scores / sorts those and then takes the top 100. 
This new trending scheme with the DB track trending score does not need this at all. It was previously done as a heuristic to make it more performant, but we do it to keep the trending consistent.